common/src/main/java/com/microsoft/identity/common/internal/ui/embeddedwebview/OAuth2WebViewClient.java

```diff
@@ -96,7 +97,7 @@ public void onReceivedHttpAuthRequest(WebView view, final HttpAuthHandler handle
 
         //TODO TelemetryEvent.setNTLM(true); after the Telemetry is finished in common.
         final NtlmChallengeHandler ntlmChallengeHandler
-                = new NtlmChallengeHandler(view, handler, host, realm, mContext, mCompletionCallback);
+                = new NtlmChallengeHandler(view, handler, host, realm, mActivity, mCompletionCallback);

```

Comment:
Consider whether [`getContext()`](https://developer.android.com/reference/android/view/View.html#getContext()) will work for you purposes and consider removing the `Activity` here